src/main/java/cambio/simulator/parsing/PatternConfigurationParser.java

```diff
@@ -103,7 +103,7 @@ public static <T extends IPatternLifeCycleHooks> T getPatternInstance(
                 strategyObject.onInitializedCompleted(model);
 
             } else {
-                System.out.printf("[Warning] No strategy information was given for a %s configuration. If no default"
+                System.out.printf("[Warning] No strategy information was given for a '%s' configuration. If no default"
```

Comment:
Although quotes here may make sense, this line was not the target of #56 (which I assume you wanted to fix with your PR), but rather https://github.com/Cambio-Project/resilience-simulator/blob/30d04a6ba4d918afcef10220c3b4d702154d2751/src/main/java/cambio/simulator/parsing/adapter/ConfigurableNamedTypeAdapter.java#L75

It's arguable, whether quotes at this location make sense, since `%s` will always be a single word name. Therefore, there is no need for citing/marking the start and end. However, I guess we can keep them.

Reply:
I get it @LitschiW. Made the changes at the appropriate place. And sorry for the delay in getting back to you.